config-connector/solutions/iam/kpt/pubsub-topic/README.md

```diff
@@ -0,0 +1,39 @@
+Pub/Sub Topic
+==================================================
+# NAME
+  pubsub-topic
+# SYNOPSIS
+  Config Connector compatible YAML files to grant a role to a particular IAM member for a PubSub topic.
+# CONSUMPTION
+  Download the package using [kpt](https://googlecontainertools.github.io/kpt/).
+  ```
+  kpt pkg get https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit.git/config-connector/solutions/iam/kpt/pubsub-topic .
+  ```
+# REQUIREMENTS
+  A Config Connector installation managing a GCP project with the Pub/Sub API enabled.
+# USAGE
+## SETTERS
+|    NAME    |        VALUE        |     SET BY      |         DESCRIPTION          | COUNT |
+|------------|---------------------|-----------------|------------------------------|-------|
+| iam-member | ${IAM_MEMBER?}      | PLACEHOLDER     | identity to grant privileges | 1     |
+| role       | roles/pubsub.editor | package-default | IAM role to grant            | 1     |
+| topic-name | allowed-topic       | package-default | name of PubSub topic         | 2     |
+
```

Comment:
Maybe add another title ## BASIC WORKFLOW under it? Just having one subtitle seems to be confusing.

Reply:
This was not addressed and not commented. @trodge can you please address this feedback or explain the rationale why you would like not to address the feedback. Thanks!

Reply:
Discussed offline and agreed that we are pulling the setters table out of usage section.